nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -21,6 +21,20 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `hardware.nvidia` has a new option `open` that can be used to opt in the opensource version of NVIDIA kernel driver. Note that the driver's support for GeForce and Workstation GPUs is still alpha quality, see [NVIDIA Releases Open-Source GPU Kernel Modules](https://developer.nvidia.com/blog/nvidia-releases-open-source-gpu-kernel-modules/) for the official announcement.
 
+- As Singularity has renamed to Apptainer to distinguish from an un-renamed fork,
+  there is now three packages of Singularity/Apptainer:
+  * `singularity-legacy`: From `hpcng/singularity`, which is the original repo and is now redirected to `apptainer/singularity`. The import path is `github.com/hpcng/singularity`.
+  * `apptainer`: From `apptainer/apptainer`, which is the new repo after renaming. The import path is `github.com/apptainer/apptainer`
+  * `singularity-ce`: From `sylabs/singularity`, which is the fork by Sylabs Inc.. The import path is `github.com/sylabs/singularity`
+
+  The package attribute `singularity` is specified as `singularity-legacy`.
+
+  The packages are now build with `buildGoModule` instead of `buildGoPackage`.
+  Building from unvendored sources is now supported through `override`,
+  while attribute `goPackagePath` is no longer provided.
+
```

Comment:
```suggestion
```
We don't need really need this and the changelog is usually long enough already.